templates/_includes/overview-footer.html

```diff
@@ -1,11 +1,7 @@
 <div class="overview-footer">
   <div class="footer-content">
-    <div class="hxRow">
-      <div class="hxCol">
-        <h2>Get in the Loop.</h2>
-        <p>Help evolve Helix by getting involved on GitHub, Invision, and Slack.</p>
-      </div>
-    </div>
+    <h2>Get in the Loop.</h2>
+    <p>Help evolve Helix by getting involved on GitHub, Invision, and Slack.</p>
```

Comment:
```html
<div class="hxRow">
  <div class="hxCol">
    <!-- content -->
  </div>
</div>
```
can be simplified to 
```html
<!-- content -->
```

Reply:
👍   thanks @CITguy

Reply:
@araiford  @CITguy  FYI, I have used the grid a bit in our start to CTUI. I would summarize that my experience determined that using the hxRow and hxCol as much as possible resulted in a more consistent padding and margin alignment across the app. I do understand that the code is redundant, and I tried to simplify. Simplification was not always successful. Thus, i had better results in certain situations by just keeping the one hxCol instead of simplifying.

Reply:
@ssalinas24, I'm going to take on the task of building out the different application and page layouts so that we can give you the best possible solution when developing your application.